test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextBuilderTests.cs

```diff
@@ -176,6 +178,9 @@ public void FillsRuntimeLibraryProperties()
             lib.Dependencies.Should().OnlyContain(l => l.Name == "System.Collections" && l.Version == "3.3.3");
             lib.ResourceAssemblies.Should().OnlyContain(l => l.Path == "en-US/Pack.Age.resources.dll" && l.Locale == "en-US");
 
+            // When ProjectModel supports path in the lock file library, this should assert "path/TO/package".
```

Comment:
I see. You are right, though I can't find what is even bringing that dependency. There shouldn't be a dependency from core-setup to CLI. It should be only the other way around.